doc/md/Installation_Instructions/Mac-OS-X-MacPorts-Installation-Instructions.md

```diff
@@ -0,0 +1,72 @@
+# MacPorts (Mac OS X), developer installation, ***experimental***
+
+<b><h3>These insturctions are not tested on Apple Silicon!</h3></b>
+
+## Prerequisites
+
+These instructions will show how to setup the environment on OSX to the point where you'll be able to clone and compile the repo by yourself, as on Linux, Windows, etc.
+
+1. Have MacPorts installed. Visit https://www.macports.org/ for more information.
+
+    * MacPorts may require a bit more setup. Such as setting up new include environments like:
+
+    ```bash
+    export C_INCLUDE_PATH="/opt/local/include"
+    export CPLUS_INCLUDE_PATH="/opt/local/include"
+    export LIBRARY_PATH="/opt/local/lib"
+    export LDFLAGS="-L/opt/local/lib"
+    export CFLAGS="-I/opt/local/include"
+    export CPPFLAGS="-isystem/opt/local/include -I/opt/local/include"
+    ```
+
+    other than just modifying PATH variable to `/opt/local/bin:/opt/local/sbin:/usr/local/bin:/usr/local/sbin:$PATH`.
+
+2. Install dependencies:
+
+    ```
+    sudo port install readline qt5 qt5-qtbase pkgconfig arm-none-eabi-gcc arm-none-eabi-binutils lua52
+    ```
+
+3. Clamp Python version for pkg-config
+
+    MacPorts doesn't handle Python version defaults when it comes to pkg-config. So even if you have done:
+
+    ```
+    sudo port install python39 cython39
+
+    sudo port select --set python python39  # this also makes calls to "python" operate on python3.9
+    sudo port select --set python3 python39
+    sudo port select --set cython cython39 
+    ```
+
+    This won't set a default python3.pc (and python3-embed.pc) under the MacPorts pkgconfig includes folder.
+
+    To fix that, follow these steps:
+
+    ```
+    cd /opt/local/lib/pkgconfig
+    sudo ln -svf python3.pc python-3.9.pc
+    sudo ln -svf python3-embed.pc python-3.9-embed.pc
+    ```
+
+4. (optional) Install makefile dependencies:
+
+    ```
+    sudo port install recode
+    sudo port install astyle
+    ```
+
+
+## Compile and use the project
+
+To use the compiled client, the only difference is that the Proxmark3 port is `/dev/tty.usbmodemiceman1`, so commands become:
+
+```sh
+proxmark3 /dev/ttyACM0  =>  proxmark3 /dev/tty.usbmodemiceman1
+```
```

Comment:
I think we should add an instruction to use ./pm3  shell script.   I see many users thinking they need to call the client direct themself with a serial port.